reference_implementations/gcp/architectures/terraform.tfvars

```diff
@@ -1,9 +1,9 @@
-project = "ai-deployment-bootcamp-430513"
-short_project_prefix = "adp-430513"
-user = "marcelo.lotif@vectorinstitute.ai"
-env = "devmarcelo"
+project = "your-alphanumeric-project-id"
+short_project_prefix = "short-prj-id"
+user = "your.user@gmail.com"
+env = "dev"
```

Comment:
I like the idea but I think it's better not to touch that file right now as the teams might already be making environments with that name a changing it might mess up their environments. It's better to do that after we revoke access to the teams.